app/services/github_client.rb

```diff
@@ -0,0 +1,26 @@
+class GithubClient
+  def initialize(login, token)
+    @login = login
+    @token = token
+  end
+
+  def is_organization_admin?(org)
```

Comment:
As I commented [above](https://github.com/education/classroom/pull/11/files#r30479861) I'd recommend using the `github_id` here.